src/index.d.ts

```diff
@@ -259,9 +259,9 @@ declare namespace preact {
 		/** Attach a hook that is invoked whenever a VNode is created. */
 		vnode?(vnode: VNode): void;
 		/** Attach a hook that is invoked immediately before a vnode is unmounted. */
-		unmount?(vnode: VNode): void;
+		unmount?(vnode: VNode): void; // TODO: What is the correct type for this parameter?
```

Comment:
These are public interfaces and so we should decide what interface to expose here at some point...